src/@next/Banner/Banner.tsx

```diff
@@ -53,43 +64,87 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
       ? iconName
       : iconByStatus || iconNameStatusMap['info'];
 
+    if (type === 'static')
+      return (
```

Comment:
Can we extract the duplicated codes into functions?

Reply:
umm which one is duplicated? because although similar, the ones with static and fixed are quite different; not only the ordering, but also it uses different styled div and the fixed ones has some additional data-XX attribute given, finally I refactor the css for fixed banner on a separate file so hopefully it is more readable

Reply:
for example the actions rendering parts